src/sentry/static/sentry/app/views/organizationIntegrations/installedIntegration.tsx

```diff
@@ -39,22 +37,6 @@ export type Props = {
 };
 
 export default class InstalledIntegration extends React.Component<Props> {
-  /**
-   * Integrations have additional configuration when any of the conditions are
-   * met:
-   *
-   * - The Integration has organization-specific configuration options.
-   * - The Integration has configurable features
-   */
-  hasConfiguration() {
```

Comment:
I am not sure why we had this originally but all first-party integrations are configurable so we don't need it.

Reply:
Yeah maybe someone thought there'd be an unconfigureable integration one day.